lib/home.dart

```diff
@@ -203,11 +199,28 @@ class _HomeState extends State<Home> {
   Widget build(BuildContext context) {
     final theme = Provider.of<ThemeModel>(context);
     final auth = Provider.of<AuthModel>(context);
-    final CupertinoTabController _controller = CupertinoTabController();
+
     if (auth.activeAccount == null) {
       return LoginScreen();
     }
 
+    switch (auth.activeAccount.platform) {
+      case PlatformType.github:
+        theme.setActiveTab(theme.startTabGh);
+        break;
+      case PlatformType.gitlab:
+        theme.setActiveTab(theme.startTabGl);
+        break;
+      case PlatformType.bitbucket:
+        theme.setActiveTab(theme.startTabBb);
+        break;
+      case PlatformType.gitea:
+        theme.setActiveTab(theme.startTabGt);
+        break;
+    }
```

Comment:
These seem should be placed in `themeModel.init()` method.

Reply:
Oh, can we get the `platform` on the theme model? Seems it could be an issue.

Reply:
Yeah that's why I added it here.

Reply:
OK. Let's optimize it late.